package.json

```diff
@@ -12,8 +12,8 @@
     "build": "rimraf ./dist && tsc -p tsconfig.prod.json --outDir ./dist",
     "formatDeclarations": "prettier --ignore-path *.js --write dist/*.d.ts",
     "prepublishOnly": "yarn test && yarn build && yarn formatDeclarations",
-    "test": "node scripts/update-test-tsconfig.js && prettier -c {bench,lib,test}/**/*.ts && tsc --project tsconfig.test.json --outDir ./testDist && rimraf ./testDist",
-    "test:fix": "node scripts/update-test-tsconfig.js && prettier --write {bench,lib,test}/**/*.ts",
+    "test": "node scripts/update-test-tsconfig.js && prettier -c {bench,lib,test}/**/*.ts && eslint && tsc --project tsconfig.test.json --outDir ./testDist && rimraf ./testDist",
```

Comment:
Feedback: `eslint` requires extra 15s (30s before & 45s now) for running test on each TypeScript version which isn't ideal

Reply:
Umm...I tried a couple more times but it was never that slow, was mostly around 35-36s.

Anyways, I've moved the display tests to separate files, reducing the no. of files we've to lint. Not sure, if it actually does improve anything, but the execution time seems to be consistently around the 35s mark. Can you try running the job a couple of times to see how it does now?

<img width="1992" alt="image" src="https://github.com/user-attachments/assets/864d60fe-5654-4968-a6cf-af9966a59736">

<br><br>

> Are you aware of any other alternatives that have better execution time? 

No, not really!

Reply:
Umm...I tried a couple more times but it was never that slow, was mostly around 37-38s.

Anyways, I've moved the display tests to separate files, reducing the no. of files we've to lint. This should slightly improve the execution time, it now seems to be consistently around the 35s mark. Can you try running the job a couple of times to see how it does now?

<img width="1992" alt="image" src="https://github.com/user-attachments/assets/864d60fe-5654-4968-a6cf-af9966a59736">

<br><br>

> Are you aware of any other alternatives that have better execution time? 

No, not really!

Reply:
I'm comparing yarn test run across all TypeScript versions in master branch (7s worst) vs this branch (13s worst), so the time difference is 6s by just running `yarn test` but I assume you have to download all dependencies that may take extra 5-7s

Reply:
Comparing usage: [your previous branch](https://github.com/ts-essentials/ts-essentials/actions/runs/12182991270/usage) 6m 9s vs 7m 12s [this branch](https://github.com/ts-essentials/ts-essentials/actions/runs/12194704372/usage), so it's over by a minute

I would say that the benefit is still worth it, so let's keep it